src/sentry/integrations/api/endpoints/integration_proxy.py

```diff
@@ -214,10 +251,10 @@ def http_method_not_allowed(self, request):
 
         response = self._call_third_party_api(request=request, full_url=full_url, headers=headers)
 
-        metrics.incr(
-            "hybrid_cloud.integration_proxy.complete.response_code",
-            tags={"status": response.status_code},
+        self._add_metric(
+            metric_name="complete.response_code",
             sample_rate=1.0,
+            tags={"status": response.status_code},
```

Comment:
@cathteng - we do have success responses recorded here. Is there another success metric you were curious about recording?

Reply:
Oh I didn't realize this existed, ignore me 🙈